_posts/2025-11-03-precise-onnx.md

```
@@ -0,0 +1,85 @@
+
+## Get the Upgrade Today!
+
+This migration ensures that the popular and powerful Precise engine remains a **stable, high-performance option** for the OpenVoiceOS community for years to come. Enjoy the easier installation, greater stability, and lower CPU usage!
+
+Update your installation and try the new **ONNX-powered Precise** for yourself!
+
+  * **New Plugin:** [`ovos-ww-plugin-precise-onnx`](https://github.com/OpenVoiceOS/ovos-ww-plugin-precise-onnx))
+  * **ONNX Models:** [`precise-lite-models`](https://github.com/OpenVoiceOS/precise-lite-models))
+
+
+---
```

markdownlint-cli2 (0.18.1)

Horizontal rule style
Expected: -----; Actual: ---

(MD035, hr-style)